ajax/libs/gitment/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "gitment",
+  "filename": "gitment.browser.js",
+  "version": "0.0.3",
+  "description": "A comment system based on GitHub Issues",
+  "author": {
+    "name": "Shiquan Sun",
+    "url": "https://github.com/imsun"
+  },
+  "keywords": [
+    "comment system",
+    "GitHub Issues"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/imsun/gitment"
+  },
+  "license": "MIT",
+  "homepage": "https://imsun.github.io/gitment/",
+  "npmName": "gitment",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "gitment.browser*"
```

Comment:
Would you please also add the pattern to grab `style/default.css`? As the README said `<link rel="stylesheet" href="https://imsun.github.io/gitment/style/default.css">` Thank you.


Memo: Although we use excluded mode in `dist`, in this case, I think use included mode is fine here. Because we just need to add `gitment.browser*` as the [README](https://github.com/imsun/gitment#1-install) said.
Thanks.